scripts/generate_embeddings.py

```diff
@@ -128,7 +157,7 @@ def got_whitespace(text: str) -> bool:
 
     # Load documents
     documents = SimpleDirectoryReader(
-        args.folder, recursive=True, file_metadata=file_metadata_func
+        args.folder, recursive=True, file_metadata=ocp_file_metadata_func
```

Comment:
What url is this metadata going to use?  Per

https://redhat-internal.slack.com/archives/C068JAU4Y0P/p1717662235856389?thread_ts=1717595166.722909&cid=C068JAU4Y0P

we want to link to the original github repo when referencing this content in a user response.

Reply:
This loads the OCP docs, with URLs pointing to https://docs.openshift.com/container-platform, just as before. The runbooks are below.

Reply:
can you split the "runbook content" commit out from the "add logic to embed the runbooks" commit?  It's a bit hard to review this PR right now because the important commit has 296 files in it

Reply:
done

Reply:
thanks!